src/components/utils/react/TableFilters.tsx

```diff
@@ -49,21 +49,18 @@ export const TableFilter = ({
             <Stack>
                 {filter.items.map((item) => (
                     <Checkbox
-                        key={filter.value + item.value}
-                        value={item.value}
-                        label={item.label}
-                        checked={
-                            filtersActive[filter.value]?.includes(item.value) ??
-                            false
-                        }
+                        key={filter.id + item.id}
+                        value={item.id}
+                        label={item.name}
+                        checked={filtersActive[filter.id]?.includes(item) ?? false}
```

Comment:
nit: `!!filtersActive[filter.id]?.includes(item)` will be shorter. But `filtersActive[filter.id]?.includes(item)` should also work fine since undefined is also an option?